comp/api/api/def/component.go

```diff
@@ -47,8 +47,52 @@ type AuthorizedSet map[string]struct{}
 // AuthorizedConfigPathsCore is the the set of authorized config keys authorized for the
 // config API.
 var AuthorizedConfigPathsCore = buildAuthorizedSet(
-	"api_key", "site", "dd_url", "logs_config.dd_url",
-	"additional_endpoints", "logs_config.additional_endpoints", "apm_config.additional_endpoints",
+	"api_key",
+	"app_key",
+	"site",
+	"dd_url",
+	"additional_endpoints",
+
+	"logs_config.dd_url",
+	"logs_config.additional_endpoints",
+
+	"apm_config.additional_endpoints",
+
+	"external_metrics_provider.api_key",
+	"external_metrics_provider.app_key",
+
+	"database_monitoring.samples.api_key",
+	"database_monitoring.samples.additional_endpoints",
+
+	"database_monitoring.metrics.api_key",
+	"database_monitoring.metrics.additional_endpoints",
+
+	"database_monitoring.activity.api_key",
+	"database_monitoring.activity.additional_endpoints",
+
+	"network_devices.metadata.api_key",
+	"network_devices.metadata.additional_endpoints",
+
+	"network_devices.snmp_traps.forwarder.api_key",
+	"network_devices.snmp_traps.forwarder.additional_endpoints",
+
+	"network_devices.netflow.forwarder.api_key",
+	"network_devices.netflow.forwarder.additional_endpoints",
+
+	"network_path.forwarder.api_key",
+	"network_path.forwarder.additional_endpoints",
+
+	"container_lifecycle.api_key",
+	"container_lifecycle.additional_endpoints",
+
+	"container_image.api_key",
+	"container_image.additional_endpoints",
+
+	"sbom.api_key",
```

Comment:
I'm seeing those `$PREFIX.api_key`, but looking at the code I don't see how this is defined, it seems the api key is more a field from entries in `additional_endpoints` actually

Reply:
You're right, thanks